lib/clas.gd

```diff
@@ -426,8 +426,7 @@ DeclareGlobalFunction( "RationalClassesSolvableGroup" );
 ##  <Func Name="CentralizerSizeLimitConsiderFunction" Arg='sz'/>
 ##
 ##  <Description>
-##  returns a function (with arguments <C>fhome</C>, <C>rep</C>, <C>cen</C>,
-##  <C>K</C>, <C>L</C>)
+##  returns a function (with arguments <C>cen</C>, <C>K</C>)
```

Comment:
I don't think we can do this, as it changes behavior of a documented and public function in a way that would break existing valid code using it (namely code using a custom "consider" function; I have used such code myself in the past)

Reply:
I've reverted this.